Atomic/syscontainers.py

```diff
@@ -926,12 +933,12 @@ def _do_checkout(self, repo, name, img, upgrade, deployment, values, destination
                     config = json.loads(config_file.read())
                 except ValueError:
                     raise ValueError("Invalid config.json file in given remote location: {}.".format(destination_path))
-                config['root']['path'] = remote_rootfs
+                config['root']['path'] = rootfs
             with open(destination_path, 'w') as config_file:
                 config_file.write(json.dumps(config, indent=4))
             # create a symlink to the real rootfs, so that it is possible
             # to access the rootfs in the same way as in the not --remote case.
-            os.symlink(remote_rootfs, os.path.join(destination, "rootfs"))
+            os.symlink(rootfs, os.path.join(destination, "rootfs"))
```

Comment:
what do you think about moving the entire `if remote_path:` block to a new function `_rewrite_rootfs(destination_path, rootfs)`?